Add instruction document

Muaad2001 opened the pull request:
README.md is added to describe how to start the website for development

The READEME.md includes:
1. a title "Movies Website for COSC 381"
2. a section for steps to start the website, which talks about:
    a. virtual environment
    b. pip install dependencies
    c. set up the environment variables for the flask app
    d. initialize the app's database and start the flask app

issue #1

Muaad2001 committed:
doc: add README.md for starting the website

Add step by step instructions for starting the website:
1, set up virtual environment and install dependencies
2. set up the environment variables for the flask app
3. initialize the database and start the flask app
4. add the correct URL to check the webpage

issue #1

Muaad2001 committed:
doc: Format README.md and add more descriptions

Added more decriptions for each step.
Specified the datafile path.
Added a title and a section format.
Specified that this instruction is for development.

Issue #1

pulpopus commented:
Steps could point to specific files or commands, like "install dependencies from requirements.txt"

Muaad2001 committed:
doc: Add "install dependencies from requirements.txt"

1. specific line to show were requirements.txt is being installed
2. code segment on how to install requirements.txt is provided

issue #1

Muaad2001 commented:
The commit 5684a3a solved the suggestion above
